### ISSUE TITLE
Emit logging-not-lazy for string concatenated via + operator

### ISSUE THREAD

**Issue by rogalski** [PyCQA/pylint#1479]
### Steps to reproduce
Run Pylint on file:
```
import logging
log = logging.getLogger()
var = "123"
log.info("Var: " + var)
```
### Current behavior
`logging-not-lazy` not emitted.

### Expected behavior
`logging-not-lazy` emitted.

### pylint --version output
master

### TODO
https://github.com/PyCQA/pylint/blob/master/pylint/checkers/logging.py#L176
- [ ] Detect whether `node.args[format_pos]` is a `BinOp` where `.op` is an addition and at least one of operands is a literal string
- [ ] Emit a message in this case
- [ ] Write unit tests
- [ ] Write ChangeLog and What's New entries

**Comment by hippo91** [comment]
I took a look at the unittest and there is the case where ``node.args[format_pos]`` is a ``BinOp`` and ``.op`` is an addition and both operands are literal strings. It is written that no message should be emitted.
Is it really the case?

`# Statements that should not be flagged:`
`renamed_logging.warn('%s' + ' the rest of a single string')`
